app/lib/actions/pulp/repository/presenters/docker_presenter.rb

```diff
@@ -31,7 +31,7 @@ def humanized_details
 
             if content_started?(download_details)
               if items_total(download_details) > 0
-                ret << (_("New images: %{count}.") % {:count => count_summary})
+                ret << (_("New manifests: %{count}.") % {:count => count_summary})
```

Comment:
The pulp code appears to key the download count off of blobs instead of manifests.